todo_app/app.py

```diff
@@ -37,21 +40,36 @@ def create_app():
     app = Flask(__name__)
     app.config.from_object(Config())
 
+    app.logger.setLevel(app.config['LOG_LEVEL'])
+
     login_manager = LoginManager()
 
     @login_manager.unauthorized_handler
     def unauthenticated():
         state = string.digits
         stateString = ''.join(random.choice(state) for i in range(20))
         session['user-state'] = stateString
+        app.logger.info("User is unauthenticated")
         return redirect('https://github.com/login/oauth/authorize?client_id=' + app.config['CLIENT_ID'] + '&state=' + stateString)
 
     @login_manager.user_loader
     def load_user(user_id):
+        app.logger.info("User is authenticated and their role is %s", User(user_id).role)
```

Comment:
You should add User ID to your logging as per submission checklist _Your app has useful logging, using sensible log levels and including information like **user and database ids** where appropriate_